java/code/src/com/redhat/rhn/domain/contentmgmt/FilterCriteria.java

```diff
@@ -69,7 +69,9 @@
     static {
         validCombinations.add(Triple.of(PACKAGE, CONTAINS, "name"));
         validCombinations.add(Triple.of(PACKAGE, EQUALS, "nevr"));
+        validCombinations.add(Triple.of(PACKAGE, GREATEREQ, "nevr"));
         validCombinations.add(Triple.of(PACKAGE, EQUALS, "nevra"));
+        validCombinations.add(Triple.of(PACKAGE, GREATEREQ, "nevra"));
```

Comment:
Does it make sense to "only" implement "greater equal" ?
Why not just implement all variants in one shot?

Reply:
Yes that was my plan :)
I am just not satisfied with the way I do the comparison down below yet.
The way we currently save the value (nevra) in the db it's hard to exract the evr part so I might need to do some changes there (still thinking about solutions)